pyproject.toml

```diff
@@ -32,6 +32,7 @@ pyicu = [
 ]
 natsort = "^8.4.0"
 univers = "30.12.1"
+cyclonedx-bom = "5.0.0"
```

Comment:
why do you need this package? Is not the `cyclonedx-python-lib` sufficient?

Reply:
Removed